frontend/app/work_packages/controllers/dialogs/columns.js

```diff
@@ -41,7 +41,7 @@ module.exports = function($scope, $filter, columnsModal, QueryService, WorkPacka
   // Available selectable Columns
   vm.promise = QueryService.loadAvailableColumns()
     .then(function(availableColumns){
-      vm.availableColumns = availableColumns;
+      vm.availableColumns = QueryService.getAvailableUnusedColumns();
```

Comment:
Still have to ask:

The code reads: 

1. load available columns
2. then disregard those columns (given as parameter)
3. and take available unused columns instead

I guess the answer will be something like:
`loadAvailableColumns` does the http request, while `getAvailableUnusedColumns` does no networking, but returns the right thing...

Still it looks weird, to have this single most return value of `loadAvailableColumns` ignored...